# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/nikolaus/gpart/errno.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/disku.c.org	Thu May 15 18:31:53 2003
+++ ./src/disku.c	Thu May 15 18:32:02 2003
@@ -19,6 +19,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <sys/ioctl.h>
+#include <errno.h>
 #include "gpart.h"
 
 #if defined(__linux__)
@@ -27,7 +28,6 @@
 #endif
 
 #if defined(__FreeBSD__)
-#include <errno.h>
 #include <sys/disklabel.h>
 #endif
 
--- ./src/l64seek.c.orig	2003-05-20 11:33:12.000000000 +0200
+++ ./src/l64seek.c	2003-05-20 11:33:33.000000000 +0200
@@ -15,6 +15,7 @@
  */
 
 #include "l64seek.h"
+#include <errno.h>
 
 
 #define OSTACKLEN	16
